src/app/android/app/build.gradle

```diff
@@ -52,8 +52,8 @@ android {
 
     defaultConfig {
         applicationId "com.nventive.internal.flutterapptemplate"
-        minSdkVersion 30
-        targetSdkVersion 34
+        minSdkVersion 31
```

Comment:
Should we also update iOS? Think we should support 15 and up

Reply:
You're right, for iOS it's 16+